cli/internal/runcache/runcache.go

```diff
@@ -155,8 +153,9 @@ func (tc *TaskCache) RestoreOutputs(ctx context.Context, prefixedUI *cli.Prefixe
 		}
 	} else {
 		// If no outputs have changed, that means we have a local cache hit.
-		cacheStatus.Local = true
-		prefixedUI.Warn(fmt.Sprintf("Skipping cache check for %v, outputs have not changed since previous run.", tc.pt.TaskID))
+		// We will still check to see whether the cache exists, so we can get the duration.
+		cacheStatus = tc.rc.cache.Exists(tc.hash)
+		prefixedUI.Warn(fmt.Sprintf("Configured outputs are already in place for %s, skipping cache check", tc.pt.TaskID))
```

Comment:
```suggestion
		prefixedUI.Warn(fmt.Sprintf("Cache hit for %s; Configured outputs are already in place, skipping expanding cache into output directories", tc.pt.TaskID))
```
Pretty wordy, but based on your message in that channel, seems more communicative. It just feels weird to me to say "skipping cache check", because if this message is supposed to communicate there was a cache hit...why would we check a cache in that case?

Reply:
"Skipping cache check" might be revealing the internal implementation, but it _is_ more correct than what you've proposed here. It's true that expanding the cache is skipped, but it's also true that we didn't even check the cache to see if there is a hit or not. If you deleted the cache, you'd still get this message.

I'll think about your suggestion for a bit though, thanks!

Reply:
> "Skipping cache check" might be revealing the internal implementation, but it _is_ more correct than what you've proposed here. It's true that expanding the cache is skipped, but it's also true that we didn't even check the cache to see if there is a hit or not. If you deleted the cache, you'd still get this message.
> 
> I'll think about your suggestion for a bit though, thanks!

😆 I think that's where the terminology gets confusing. "Cache hit" as an outside dev to me could mean:

1. The inputs haven't changed and therefore the hit/miss is immaterial, we don't even need to look at the cache
2. The inputs haven't changed and therefore if we were to check the cache, we'd definitely get a hit

If I'm understanding, you are saying 1 is really what we want to communicate.

Reply:
it may not be what we _want_ to communicate to an outside dev, but what's happening is two things:

1. you ran this task before, and we know you wrote outputs XYZ. We've been watching those outputs since then, and we _know_ they haven't changed.
2. we also know your _inputs_ didn't change, because we are starting with the same task hash.
3. so we know we can skip the cache check and the cache restoration

Reply:
There's enough to talk about that I'm moving it to its own PR